snippet try
    try{

    }catch(Exception e){

    }finally{

    }
